application/helpers/export_helper.php

```diff
@@ -2746,12 +2746,12 @@ function tsvSurveyExport($surveyid)
         $defaultvalues_data = array();
     }
     // insert translations to defaultvalues_datas
+    $defaultvalues_datas = [];
     if (array_key_exists('defaultvalue_l10ns', $xmlData)) {
         $defaultvalues_l10ns_data = $xmlData['defaultvalue_l10ns']['rows']['row'];
         if (!array_key_exists('0', $defaultvalues_l10ns_data)) {
```

Comment:
https://github.com/LimeSurvey/LimeSurvey/pull/4181/files#diff-1aecd900cde469d4c2ba64ab989062a075ed3ff9378180051d812b127a929dfeR2839 

!array_key_exists('0', $myArray) test are the same than array_key_exists('id', $myArray) :)

Reply:
Sorry, not following here.